II.pass-tests/08-get-average/get-average.js

```diff
@@ -1,3 +1,22 @@
 // the input is an array of numbers and strings
 // return the average of all the numbers
-// be sure to exclude the strings
\ No newline at end of file
+// be sure to exclude the strings
+function average(numbers){
+    var sum = 0;
+    const arrayNumbers = numbers.filter(number => {
+        return typeof number === "number";
+    } )
+    arrayNumbers.forEach(number => {
```

Comment:
is there a way to use `reduce` to improve the code? ;)

Reply:
@lpalma I figured I can use the reduce method, will be pushing the update shortly